site/topics/origin/origin.rst

```diff
@@ -209,6 +209,40 @@ Turing Completeness in Simple Systems
 The Invisible Hand
 ==================
 
+* Beware of *teleology*
+
+    * Explaining phenomena in terms of the purpose they serve rather than of the cause by which they arise
+
+
+* Virtually nothing within the universe exists for a purpose
+
+    * The exceptions being things like human created things
+
+
+* Things only exist because they don't not
+* Evolution has no intelligence guiding it
+* Yet, things seem to *work*
+* But remember, they works, because if it did not, it would exist
```

Comment:
english exists, but not here

Reply:
selecting a single line is hard. line 225 doesn't read well

Reply:
they works

Reply:
![image](https://github.com/jameshughes89/cs4XX-EvolutionaryComputation/assets/5175174/1a20f5dc-4150-49f1-829c-7d1f3d5c3e9a)